containers/src/Kokkos_DynRankView.hpp

```diff
@@ -1252,10 +1329,17 @@ as_view_of_rank_n(
                 std::is_same_v<decltype(layout), Kokkos::LayoutRight> ||
                 std::is_same_v<decltype(layout), Kokkos::LayoutStride>) {
     for (int i = N; i < 7; ++i)
-      layout.dimension[i] = KOKKOS_IMPL_CTOR_DEFAULT_ARG;
+      layout.dimension[i] = 1;  // KOKKOS_IMPL_CTOR_DEFAULT_ARG;
```

Comment:
I thought `KOKKOS_IMPL_CTOR_DEFAULT_ARG` isn't defined as `1` but as `~std::size_t(0)`

Reply:
Let me check this change might actually have been unnecessary and was part of some trial and error approach to fixing stuff.

Reply:
fixed